### PR TITLE
feat: add switch of online debug

### PR DESCRIPTION
**Pull request by starsz**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

Before this PR, we don't have a switch of the online debug.
This PR is to add a switch to control the online debug function

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

We can decide whether to enable the online debug function.

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [x] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Comment by liuxiran** [comment]
May I ask why we need a switch for online debug function  @starsz

File: api/internal/handler/route_online_debug/route_online_debug.go [hunk 1]
```diff
@@ -70,6 +71,11 @@ type Result struct {
 }
 
 func (h *Handler) DebugRequestForwarding(c droplet.Context) (interface{}, error) {
+	if !conf.OnlineDebugConfig.Enable {
+		return &data.SpecCodeResponse{StatusCode: http.StatusBadRequest},
+			fmt.Errorf("should enable the online debug in managerapi")
```
Comment (nic-chen) [inline api/internal/handler/route_online_debug/route_online_debug.go:76]: In `conf.yaml` should be better

File: api/test/docker/manager-api-conf.yaml [hunk 2]
```diff
@@ -25,6 +25,8 @@
       - 172.16.238.10:2379   # ips here are defined in docker compose.
       - 172.16.238.11:2379
       - 172.16.238.12:2379
+  online_debug:
+     enable: true
```
Comment (nic-chen) [inline api/test/docker/manager-api-conf.yaml:29]: Default false

Comment (starsz) [reply api/test/docker/manager-api-conf.yaml:29]: It's test/docker/manager-api-conf.We should enable it to test online debug.

File: api/internal/handler/route_online_debug/route_online_debug.go (hunk 1 above)
Comment (starsz) [reply api/internal/handler/route_online_debug/route_online_debug.go:76]: `conf.yaml` ? Don't know about it.

Comment (nic-chen) [reply api/internal/handler/route_online_debug/route_online_debug.go:76]:
```suggestion
			fmt.Errorf("should enable the online debug in conf.yaml")
```

File: api/test/docker/manager-api-conf.yaml (hunk 2 above)
Comment (nic-chen) [reply api/test/docker/manager-api-conf.yaml:29]: ok

File: api/test/docker/manager-api-conf.yaml [hunk 3]
```diff
@@ -25,6 +25,8 @@ conf:
...
```
Comment (liuxiran) [inline api/test/docker/manager-api-conf.yaml:29]:
```suggestion
    enable: true
```

**Comment by juzhiyuan** [comment]
I have replied in the mailing list, do we really need a Switcher to fix this issue?

**Pull request closed without merging**